app/services/user_sign_up.rb

```diff
@@ -0,0 +1,46 @@
+class UserSignUp
+  attr_reader :error_message
+
+  def initialize(user)
+    @user = user
+  end
+
+  def sign_up(stripe_token, invitation_token)
```

Comment:
This method signature requires two arguments. Using an options hash does not. Just something to be aware of. You could also set a default value of `nil` for `invitation_token`.